crates/eips/src/eip1559/basefee.rs

```diff
@@ -7,17 +7,17 @@ use crate::eip1559::constants::{
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct BaseFeeParams {
     /// The base_fee_max_change_denominator from EIP-1559
-    pub max_change_denominator: u64,
+    pub max_change_denominator: u128,
     /// The elasticity multiplier from EIP-1559
-    pub elasticity_multiplier: u64,
+    pub elasticity_multiplier: u128,
```

Comment:
This needs serde attr

Reply:
it's not part of rpc, but we should do hex here yes